checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitVisitor.java

```diff
@@ -6,6 +6,7 @@
 import com.sun.source.tree.ArrayTypeTree;
 import com.sun.source.tree.AssertTree;
 import com.sun.source.tree.BinaryTree;
+import com.sun.source.tree.BindingPatternTree;
```

Comment:
This class doesn't exist on Java < 16. So you have to use
[isBindingPatternTree](https://github.com/eisop/checker-framework/blob/4935c5d74e2a26a2d52859075a10ac1f9fa42aad/javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java#L2366) and then [BindingPatternUtils#getVariable](https://github.com/eisop/checker-framework/blob/4935c5d74e2a26a2d52859075a10ac1f9fa42aad/javacutil/src/main/java/org/checkerframework/javacutil/TreeUtilsAfterJava11.java#L55).